pennylane/transforms/qcut.py

```diff
@@ -210,3 +213,121 @@ def tape_to_graph(tape: QuantumTape) -> MultiDiGraph:
             order += 1
 
     return graph
+
+
+# pylint: disable=too-many-branches
+def contract_tensors(
+    tensors: Sequence,
+    communication_graph: MultiDiGraph,
+    prepare_nodes: Sequence[Sequence[PrepareNode]],
+    measure_nodes: Sequence[Sequence[MeasureNode]],
+    use_opt_einsum: bool = False,
+):
+    """Contract tensors according to the edges specified in the communication graph.
+
+    This operation is differentiable. The ``prepare_nodes`` and ``measure_nodes`` arguments are both
+    sequences of size ``len(communication_graph.nodes)`` that describe the order of indices in the
+    ``tensors`` with respect to to the :class:`~.PrepareNode` and :class`~.MeasureNode` edges in the
+    communication graph.
```

Comment:
From what I can tell, this sentence mostly re-states information that is already present in the `Args:` section?

I suggest moving any required info from here down to the corresponding `Args:`, to avoid duplication/extra maintenance.

Instead, this sentence could be retooled to provide some more info on what this function does/when one would want to use it?

Reply:
Thanks, I have elaborated: f6156dbea448b4a8cca567ec02b862642a69dbb1